packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -247,6 +247,13 @@ export interface CallAdapterCallManagement {
    * @public
    */
   startCall(participants: string[], options?: StartCallOptions): Call | undefined;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /**
+   * Start the call.
+   * @param participants - An array of {@link @azure/communication-common#CommunicationIdentifier} to join
```

Comment:
```suggestion
   * @param participants - An array of {@link @azure/communication-common#CommunicationIdentifier} to be called
```

Technically, it's not a join flow but an outbound call that needs to be answered by the call receiver...